src/index.ejs

```diff
@@ -66,7 +66,14 @@
               ]
             },
             dataCenter: '{!= info.general.dataCenter !}',
-            kubernetesVersions: [],
+            kubernetesVersions: {
+              {! Object.values(JSON.parse(info.general.kubernetesVersions)).map((versionInfo) => { !}
+                  '{!= versionInfo.minorVersion !}': {
+                    minorVersion: '{!= versionInfo.minorVersion !}',
```

Comment:
Isn't this field write-only now?

Reply:
What do you mean? `versionInfo` is a value of the map created by `JSON.parse(info.general.kubernetesVersions)`

Reply:
I mean `minorVersion` field.

Reply:
This file is a template, and it uses `ejs` as a templating language. Here is where the `kubernetesVersions` map is generated, and for every value of the map parsed from `JSON.parse(info.general.kubernetesVersions)`, we would render a new entry into our `kubernetesVersions` map.

In the output code, we would have a similar object:
```typescript
...

kubernetesVersions: {
	"1.15": {
		minorVersion: "1.15",
		eolDate: "2020-03-12,
	},
	"1.16": {
		minorVersion: "1.16",
		eolDate: "2020-07-12,
	},
	"1.17": {
		minorVersion: "1.17",
		eolDate: "2020-11-12,
	},
},

...
```

So setting `minorVersion` here is just the same as setting the value of any random key in a JS object. Nothing is read/write-only